docs/tests-load.rst

```diff
@@ -0,0 +1,38 @@
+Load Testing with Locust
+========================
+
+We use Locust for load-testing MDN. We write locust test files for different
+behavior patterns on MDN, and run the tests to simulate those behaviors
+against other environments.
+
+.. note:: **DO NOT RUN LOCUST TESTS AGAINST PRODUCTION**
+
+Installing
+----------
+
+See `Installation Docs for Locust
+<http://docs.locust.io/en/latest/installation.html>`_. (Notice there
+are some platform-specific instructions.)
+
+Running Tests
+-------------
+
+.. note:: **DO NOT RUN LOCUST TESTS AGAINST PRODUCTION**
+
+1. Start locust with one of our test files and our target host::
+
+    locust -f tests/load/smoke.py --host=https://developer.allizom.org
```

Comment:
I think I'd like to wrap that in a Makefile and hardcode the host so that we don't accidentally use prod.